content/contribute/index.md

```diff
@@ -40,94 +42,110 @@ The biggest and most active repositories we have today are:
 - https://github.com/libp2p/js-libp2p
 
 If you want to start contributing to the core of IPFS, those repositories are
-a great start. But the "Help Wanted" label exists in all of our repositories
-(in the Github organizations IPFS / libp2p / IPLD / Multiformats)
+a great place start. But the “Help Wanted” label exists in all of our
+repositories across the Github organizations
+[IPFS](https://github.com/ipfs),
+[libp2p](https://github.com/libp2p),
+[IPLD](https://github.com/libp2p), and
+[Multiformats](https://github.com/multiformats).
+
 
 ### Documentation
 
 Again, IPFS is a huge project and undertaking. With lots of code comes the need
-of having lots of good documentation too! However, this is currently are area
-we're severly lacking enough manpower to make great. If writing technical documentation is your area, we'd love your help!
+for lots of good documentation! However, we need a lot more help to write the
+awesome docs the project needs. If writing technical documentation is your area,
+we’d absolutely love your help!
+
+The best place to get started is by looking through the Github Issues at:
+https://github.com/ipfs/docs
```

Comment:
Should link directly to the issues, possibly with the label "Help Wanted" already preselected, like this: https://github.com/ipfs/docs/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22 (currently empty)